components/raftstore/src/store/async_io/write.rs

```diff
@@ -0,0 +1,839 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
```

Comment:
Can you add module docs to explain how data are processed in different structure of this file?